src/mono/wasm/runtime/types/internal.ts

```diff
@@ -133,17 +134,20 @@ export type LoaderHelpers = {
     getPromiseController: <T>(promise: ControllablePromise<T>) => PromiseController<T>,
     assertIsControllablePromise: <T>(promise: Promise<T>) => asserts promise is ControllablePromise<T>,
     mono_download_assets: () => Promise<void>,
-    resolve_asset_path: (behavior: AssetBehaviours) => AssetEntryInternal,
+    resolve_asset_path: (behavior: AssetBehaviors) => AssetEntryInternal,
```

Comment:
I know `AssetBehaviours` name is not great, but is it worth breaking API change ?

Reply:
My thoughts are that we are now breaking a lot of things in terms of TS, but I can revert this one if you prefer

Reply:
I'm not sure